uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProviderConfiguration.java

```diff
@@ -75,6 +75,8 @@
     public static final String GIT_ENV_KEY_INIT = "init";
     public static final String GIT_ENV_KEY_MIRROR = "mirror";
     public static final String SCHEME = "git";
+    public static final String GIT_SSH_CIPHERS = "org.uberfire.nio.git.ssh.ciphers";
```

Comment:
Please don't forget to update the property name in kie-docs

Reply:
Yep, https://github.com/kiegroup/kie-docs/pull/1346 :)